oasislmf/model_preparation/sum_inputs.py

```diff
@@ -167,21 +123,12 @@ def group_by_oed(summary_map_df, exposure_df, oed_col_group):
     :rtype: list
     """
 
-    # Guard check that set(column_set) < set(exposure_col_df.columns.to_list())
-    # --> OasisExecption if not
-
-
-    # -> Check is column_set subset of summary_map_df.columns.to_list()
-
-
-    # N -> import missing from OED
     exposure_cols = [c.lower() for c in oed_col_group if c not in summary_map_df.columns.to_list()]
     mapping_cols = [SOURCE_IDX['loc']] + [c.lower() for c in oed_col_group if c in summary_map_df.columns.to_list()]
 
     summary_group_df = summary_map_df[mapping_cols]
     if not exposure_cols == []:
         exposure_col_df = exposure_df[exposure_cols]
```

Comment:
Same as above - better to use

    exposure_col_df = exposure_df.loc[:, exposure_cols]